programmers/문자열-다루기-기본.js

```diff
@@ -0,0 +1,6 @@
+function solution(s) {
+  let result = parseInt(s);
```

Comment:
자바스크립트는 항상 최우선적으로 const로 변수 타입 설정하고, 그 이후에 값이 변하면 let으로 수정하기
```suggestion
  const result = parseInt(s);
```